src/utils/paths.rs

```diff
@@ -20,6 +20,7 @@ pub const HASHMAP: [&'static str; 5] = ["std", "collections", "hash", "map", "Ha
 pub const HASH: [&'static str; 2] = ["hash", "Hash"];
 pub const IO_PRINT: [&'static str; 3] = ["std", "io", "_print"];
 pub const LINKED_LIST: [&'static str; 3] = ["collections", "linked_list", "LinkedList"];
+pub const MEM_FORGET: [&'static str; 3] = ["core", "mem", "forget"];
```

Comment:
I don't quite understand why the path to this is `core::mem::forget` when I'm explicitly using `std::mem::forget` in the tests. Is this just an alias? Can we rely on this?